Make all static pages mobile-friendly

stvstnfrd opened the pull request.

stvstnfrd committed:
Tweak footer logo display

stvstnfrd committed:
Cleanup static pages

stvstnfrd commented on lagunita/lms/static/sass/partials/base/custom_overrides/_header.scss:
```diff
@@ -51,3 +51,44 @@
         }
     }
 }
+section.container {
```
Oh, I need to move this somewhere better.
I just stuck it here to get started.

caesar2164 replied:
maybe add a new `_mobile.scss`?

caesar2164 commented on lagunita/lms/templates/static_templates/contact.html:
```diff
@@ -13,13 +13,9 @@
+    <section class="container contact">
         <section class="vision">
             <section class="message left" id="contact">
-                <div class="photo">
```
why are removing the dish photo?

stvstnfrd replied:
Eh, why not?
The about page looks nice spruced up w/ an image, but this is just contact info, so one less distraction?
I mean, we could include it inline, but doesn't feel like it adds much to the page. It's cute and all, the dish for contact ;)

stvstnfrd committed:
Hide contextual help link

This is a little bit wonky for us since it links prominently to edX's
documentation. While there is helpful information there, it also makes
assumptions about edX's deployment, mentions features we aren't using,
etc., so I think it's best to keep hiding this.

Also, this added header item causes the page to break more quickly on
narrower windows, so this mitigates somewhat.

caesar2164 commented on lagunita/lms/static/sass/partials/base/custom_overrides/_footer.scss:
```diff
@@ -5,7 +5,7 @@
             width: $footer-logo-width;
             float: left;
-            margin: 0 25px 0 0;
+            margin: 0 15px 15px 0;
```
I agree with this change :)

caesar2164 commented on lagunita/lms/static/sass/partials/base/_overrides.scss:
```diff
@@ -22,3 +22,5 @@ $footer-logo-width: 100px;
     display: none;
 }
+
+@import 'custom_overrides/_mobile.scss';
```
Isn't it nice getting to keep this file so nice and clean?

caesar2164 reviewed (approved):
👍
